RELEASE-NOTES.txt

```diff
@@ -1,3 +1,9 @@
+1.1.1
+=====
+* Updated wp-now to fix missing curl and webp issues
+* Improved translation support and updated translations for multiple languages
+* Resolved server stop issue by adjusting force-stop functionality.
```

Comment:
should we add a mention of the language switcher that we introduced?

Reply:
Added one more item. 
What do you think @katinthehatsite

Reply:
We could also potentially include: https://github.com/Automattic/studio/pull/495
As well as mention this improvement: https://github.com/Automattic/studio/issues/132

Reply:
Isn't that too minor?

Reply:
I have only done a release once so let's maybe ask someone more experienced - @wojtekn what do you think?

Reply:
Agreed, it makes sense to add a language switcher. We can add an item saying we improved compatibility with WooCommerce, too.